resources/benchmark-runner.mjs

```diff
@@ -255,14 +312,17 @@ export class BenchmarkRunner {
     }
 
     async _recordTestResults(suite, test, syncTime, asyncTime, unused_height, testDoneCallback) {
-        const suiteResults = this._measuredValues.tests[suite.name] || { tests: {}, total: 0 };
-        const total = syncTime + asyncTime;
-        this._measuredValues.tests[suite.name] = suiteResults;
-        suiteResults.tests[test.name] = { tests: { Sync: syncTime, Async: asyncTime }, total: total };
-        suiteResults.total += total;
-
-        if (this._client?.didRunTest)
-            await this._client.didRunTest(suite, test);
+        // Skip reporting updates for the warmup suite.
+        if (suite !== WarmupSuite) {
```

Comment:
Can we instead have an early return?:
if (suite === WarmupSuite) {
    testDoneCallback();
    return;
}

Reply:
Done.